### PR TITLE
Modifications for gfsda.v16.3.0 branch

### PR DESCRIPTION
**Pull request by emilyhcliu**
Change the following to gfsda.v16.3.0 branch:
A. /scripts/exglobal_atmos_analysis.sh:
(1) set cao_check to true
(2) set ta2tb to true
(3) set rmesh(4)=50 for AVHRR and VIIRS (NSST application)
(4) point CRTM cloud coefficient file to GFDL one

B. /modulefiles/gsi_common.lua:
(1) change crtm_ver from 2.3.0 to 2.4.0

C. Create specific fix branch which include fix files for global application only (DA_GFSv16.3.0_global_only)

### CODE REVIEW

**Comment by emilyhcliu** [comment]
Tagging @ADCollard @HaixiaLiu-NOAA @XuLi-NOAA 
Please take a look at these changes.  Thanks.

**Review by HaixiaLiu-NOAA — APPROVED** [review]
changes look good to me.

**Comment by emilyhcliu** [comment]
@MichaelLueken-NOAA We have some changes in GSI fix files.
We created a subdirectory under /GSI/fix/gfsv16_historical/gfsv16.3

There are two historical satinfo files and two historical convinfo files.
They are specifically updated for the gfsv16.3 paralle testing (the parallel experiment starts from 20211015 18z)

Could you help us add this new directory into the gfsv16_historical?
Thanks a lot.

**Comment by RussTreadon-NOAA** [comment]
Both `release/gfsda.v16.3.0` and `release/gfsda.v16.3.0_emily` currently point at `fix @ 99f147c`.  This is not correct for implementation.  `fix @ 99f147c` contains more than just global GSI fix files.   NCO only wants global GSI fix files in the implementation package. 

For example, the fix associated with WCOSS2 tag `gfsda.v16.2.0` is `DA_GFSv16.1.8_global_only`   We need to create a trimmed down (global only) `fix` for the GFS v16.3 implementation.

**Comment by emilyhcliu** [comment]
@ADCollard and @HaixiaLiu-NOAA 
I created a subdirectory under /GSI/fix/gfsv16_historical/gfsv16.3

There are two historical satinfo files and two historical convinfo files.
They are specifically updated for the gfsv16.3 paralle testing (the parallel experiment starts from 20211015 18z)

The global-workflow will these the convinfo and satinfo according to the cycle date/time from this directory.

Please see the corresponding PR in global-workflow related to this PR
[global-workflow PR #876](https://github.com/NOAA-EMC/global-workflow/pull/876)

**Comment by emilyhcliu** [comment]
> Both `release/gfsda.v16.3.0` and `release/gfsda.v16.3.0_emily` currently point at `fix @ 99f147c`. This is not correct for implementation. `fix @ 99f147c` contains more than just global GSI fix files. NCO only wants global GSI fix files in the implementation package.
> 
> For example, the fix associated with WCOSS2 tag `gfsda.v16.2.0` is `DA_GFSv16.1.8_global_only` We need to create a trimmed down (global only) `fix` for the GFS v16.3 implementation.

@RussTreadon-NOAA Thanks for pointing out the issue in the fix file.  I will use the gfsda.v16.2.0 as an example to create a proper GSI fix for v16.3 implementation.

**Comment by MichaelLueken** [comment]
@emilyhcliu Fix submodule branch `DA_GFSv16.3.0_global_only` was created from `rev2 @ 99f147c`.  To add the `gfsv16.3` subdirectory to `DA_GFSv16.3.0_global_only` and update the fix submodule revision in `release/gfsda.v16.3.0` -

In release/gfsda.v16.3.0_emily:
 * git remote update
 * cd fix
 * git checkout DA_GFSv16.3.0_global_only
 * Add the gfsv16.3 subdirectory to gfsv16_historical.
 * git add gfsv16_historical/gfsv16.3
 * git commit
 * Remove fix files not used with global.  I recommend the above commit in case you make a mistake while removing fix files, you can back out (git reset --hard HEAD~1) without removing the new subdirectory.
 * git commit
 * git push origin DA_GFSv16.3.0_global_only
 * cd ..
 * git add fix
 * git commit
 * git push origin release/gfsda.v16.3.0_emily

Once the changes in this PR are merged to `release/gfsda.v16.3.0`, the fix submodule revision will be updated to  `DA_GFSv16.3.0_global_only`.

**Comment by emilyhcliu** [comment]
@MichaelLueken-NOAA Thanks for your detailed instruction on how to create the global only fix directory under gfsda.v16.3.0
I had followed your instruction and update the release/gfsda.v16.3.0 fix with the DA_GFSv16.3.0_global_only.

Tagging @HaixiaLiu-NOAA @ADCollard  @RussTreadon-NOAA

**Comment by emilyhcliu** [comment]
@HaixiaLiu-NOAA @ADCollard @MichaelLueken-NOAA @RussTreadon-NOAA 
If there is no objection to the changes made in this PR:
 fix                               
 modulefiles/gsi_common.lua         
 scripts/exglobal_atmos_analysis.sh 

I will merge this branch back to release/gfsda.v16.3.0

Please give me an OK or let me know if you have any questions.
Thanks.

**Review by ADCollard — APPROVED** [review]
Looks good to me

**Comment by MichaelLueken** [comment]
> @MichaelLueken-NOAA Thanks for your detailed instruction on how to create the global only fix directory under gfsda.v16.3.0 I had followed your instruction and update the release/gfsda.v16.3.0 fix with the DA_GFSv16.3.0_global_only.

@emilyhcliu I just checked the changed files in this PR and I don't see the update to fix.  I do see that the `DA_GFSv16.3.0_global_only` submodule branch has had the `gfsv16.3` subdirectory added (`@ 00fe93`) and the fixed files were pruned for global only (`@ 9eed37`), but it doesn't seem as though these modifications made it back into `release/gfsda.v16.3.0_emily`.  Please check if you haven't pushed your `release/gfsda.v16.3.0_emily` branch with the updated fix files as well.

**Comment by emilyhcliu** [comment]
@MichaelLueken-NOAA I did the following after I pruned the files in the fix directory:

```
git push origin DA_GFSv16.3.0_global_only
cd ..
git add fix
git commit
git push origin release/gfsda.v16.3.0_emily
```

**Comment by XuLi-NOAA** [comment]
Emily,

rmesh(4)=50.0 is the final decision?

There seems to be no final decision yet.

Xu

**Pull request merged**
